db/migrate/20160111013907_create_option_sets.rb

```diff
@@ -0,0 +1,11 @@
+class CreateOptionSets < ActiveRecord::Migration
+  def change
+    create_table :option_sets do |t|
+      t.references :division, index: true, null: false, foreign_key: true
+      t.string :model_type
```

Comment:
Ah, yeah, I feel silly right now. This is what I get for doing code reviews late in the day. I need to think more about how this works.